scheduler/src/cook/compute_cluster.clj

```diff
@@ -46,21 +46,20 @@
              [?c :compute-cluster/mesos-framework-id ?framework-id?]]
            unfiltered-db compute-cluster-name framework-id)
         materialized-query-result (into [] query-result)]
-    materialized-query-result))
+    (first materialized-query-result)))
```

Comment:
I should have mentioned this earlier, but this returns an id, not an entity right? If so, can we clean up the name and docstring?

Also, now that you're calling `first` you can probably drop `materialized-query-result` and just call `(first query-result)`

Reply:
Sounds good. :) Doing now.

Reply:
Looks like `materialized-query-result` is still there :(

Reply:
Oops. Didn't put a comment in. I tried the change, but it didn't work, so I reverted it.

Reply:
Ok, it does work now without materializing.